<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormEquipamientoBase</class>
 <widget class="QWidget" name="FormEquipamientoBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="LTitulo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Dato del equipamiento&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="LDEs">
       <property name="text">
        <string>Descripcion:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LEDescripcion"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="LMarca">
         <property name="text">
          <string>Marca:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ECBModeloEquipamiento" name="CBMarca">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="LModelo">
         <property name="text">
          <string>Modelo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LEModelo"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="LCliente">
       <property name="text">
        <string>Cliente o Dueño:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ECBClientes" name="CBCliente">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="LNumSerie">
         <property name="text">
          <string>Nº de Serie:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LENumSerie"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="CkBGarantia">
       <property name="text">
        <string>En garantía</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Factura de compra:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="LFacturaCompra">
         <property name="text">
          <string>#0000-00000</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Fecha de compra:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="LFechaCompra">
         <property name="text">
          <string>#0000-00000</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Cantidad de elementos:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="SBCantidadElementos"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="CkBBaja">
       <property name="text">
        <string>Dado de baja</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="LObservaciones">
       <property name="text">
        <string>Observaciones:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="PTEObservaciones">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="GBHistorial">
     <property name="title">
      <string>Historial</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableView" name="TVHistorial"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ECBClientes</class>
   <extends>QComboBox</extends>
   <header>ecbclientes.h</header>
  </customwidget>
  <customwidget>
   <class>ECBModeloEquipamiento</class>
   <extends>QComboBox</extends>
   <header>ecbmodeloequipamiento.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
